### ISSUE TITLE
Model hybirds sparse tensor and dense tensor

### ISSUE THREAD

**Issue by hehaodele** [StanfordVL/MinkowskiEngine#60]
Wondering about whether it is supported to train models having both sparse tensors and dense tensors.

For example, in the task, the input data is high dimensional and sparse while the label is low dimensional and dense.  It is possible to use MinkowskiEngine to extract features from input then convert the feature into a dense tensor and use a normal PyTorch Model to make predictions?

**Comment by chrischoy** [comment]
There is `SparseTensor.dense()`, which converts a sparse tensor into a dense tensor.

The result is B x F x D1 x D2 x D3 ... where B is the batch size, F is the feature dimension, Dn's are spatial dimensions.

Would this function be helpful for your task?

**Comment by hehaodele** [comment]
Hi Chris,
Thanks for the fast response. 
`SparseTensor.dense()` is exactly what I am looking for.
